src/Models/AbstractWidget.php

```diff
@@ -89,13 +87,11 @@ public function widget( $args, $instance ) {
 	 * @param array $instance Previously saved values from database.
 	 */
 	public function form( $instance ) {
-		$title = ! empty( $instance['title'] )
-			? $instance['title']
-			: '';
+		$title = empty( $instance['title'] ) ? '' : $instance['title'];
 		?>
 		<p>
 			<label for="<?php echo esc_attr( $this->get_field_id( 'title' ) ); ?>">
-				<?php esc_html_e( 'Title:' ); ?>
+				<?php esc_html( 'Title:' ); ?>
```

Comment:
You were using a function to display translated text with no text domain.


> On Jun 1, 2016, at 8:09 AM, Adam Fenton <notifications@github.com> wrote:
> 
> In src/Models/AbstractWidget.php <https://github.com/moxie-lean/wp-widgets/pull/23#discussion_r65356289>:
> 
> >  		?>
> >  		<p>
> >  			<label for="<?php echo esc_attr( $this->get_field_id( 'title' ) ); ?>">
> > -				<?php esc_html_e( 'Title:' ); ?>
> > +				<?php esc_html( 'Title:' ); ?>
> I don't get this change... won't esc_html do nothing unless you echo it?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/moxie-lean/wp-widgets/pull/23/files/da2b2163ee75d6c22433b4221b9e2f763c6e4480#r65356289>, or mute the thread <https://github.com/notifications/unsubscribe/ADvVicSqA4X8UzfZDTMVaAGQ64fa-mepks5qHYSJgaJpZM4IlAs3>.
>